web/src/components/SideBar.tsx

```diff
@@ -41,7 +41,7 @@ const SideBar = (props: Props) => {
     } else {
       return props.errors.map((error, index) => {
         return (
-          <ListGroup.Item action className={"sidebar-item" + " " + sidebarItemClass(error)} onClick={_ => props.handleErrorSelect(error.aggregation_key)} active={ props.activeError === undefined ? false : error.aggregation_key === props.activeError.aggregation_key } key={index}>
+          <ListGroup.Item action className={"sidebar-item" + " " + sidebarItemClass(error)} onClick={_ => props.handleErrorSelect(encodeURIComponent(error.aggregation_key))} active={ props.activeError === undefined ? false : error.aggregation_key === props.activeError.aggregation_key } key={index}>
```

Comment:
do we need to url-decode it when following a deep link? (since it is read from the url and used to index the error)

Reply:
true! fixing now